src/Mvc/Mvc/src/DesignTimeMvcServiceCollectionProvider.cs

```diff
@@ -1,24 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This type was added to support the precompilation tool that was deleted in 2.0. Just happened to notice it's still around when I had to annotate it.